lib/commit-lint.ts

```diff
@@ -115,8 +115,8 @@ export class LintCommit {
                 (! this.lines[i].match(/^\s+/)) &&
                 // Allow long lines if they cannot be wrapped at some
                 // white-space character, e.g. URLs. To allow ` [1] <URL>`
-                // lines, we skip the first 10 characters.
-                this.lines[i].slice(10).match(/\s/)) {
+                // lines, we skip the first 11 characters ("Bug report ").
+                this.lines[i].slice(11).match(/\s/)) {
```

Comment:
I don't think this is a good thing to do. You could just as easily write `Bug report:` into its own line and put the overly-long line into the next line. And that allows for things like `The problem was reported at:` (which is much longer than 11 columns, but still informative and short).